SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -573,14 +573,8 @@ void ASpatialDebugger::EditorRefreshWorkerRegions()
 
 bool ASpatialDebugger::EditorAllowWorkerBoundaries() const
 {
-	// Check if multi worker is enabled.
-	UWorld* World = GetWorld();
-	check(World != nullptr);
-
-	const bool bIsMultiWorkerEnabled = USpatialStatics::IsSpatialMultiWorkerEnabled(World);
-	const bool bIsSpatialNetworkingEnabled = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
-
-	return bIsMultiWorkerEnabled && bIsSpatialNetworkingEnabled;
+	// Check if spatial networking is enabled.
+	return GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
```

Comment:
Don't we want to check for multiworker here?

Reply:
To be consistent with the run-time spatial debugger we want to show a single boundary in the editor so I needed to remove this check from here.